sdk/eventhubs/azure_messaging_eventhubs/benches/benchmarks.rs

```diff
@@ -0,0 +1,139 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+use azure_identity::DefaultAzureCredential;
+use azure_messaging_eventhubs::{models::EventData, ProducerClient, SendEventOptions};
+use criterion::{criterion_group, criterion_main, Criterion};
+use std::{env, sync::Arc};
+use tokio::runtime::Runtime;
+
+//static INIT_LOGGING: std::sync::Once = std::sync::Once::new();
+
+fn setup() {
+    // INIT_LOGGING.call_once(|| {
+    //     println!("Setting up test logger...");
+
+    //     use tracing_subscriber::{fmt::format::FmtSpan, EnvFilter};
+    //     tracing_subscriber::fmt()
+    //         .with_env_filter(EnvFilter::from_default_env())
+    //         .with_span_events(FmtSpan::NEW | FmtSpan::CLOSE)
+    //         .with_ansi(std::env::var("NO_COLOR").map_or(true, |v| v.is_empty()))
+    //         .with_writer(std::io::stderr)
+    //         .init();
+    // });
+}
+
+fn send_batch_benchmark(c: &mut Criterion) {
+    setup();
+
+    // Check if the environment variable is set thus allowing the benchmarks to run
+    if azure_core_test::TestMode::current().unwrap() != azure_core_test::TestMode::Live {
```

Comment:
I'd strongly urge `unwrap_or_default` or `unwrap_or(Playback)` or something like that. It can fail with a bad env var value. I suppose at least it fails in the way we want, but won't print your informative message below it.

Reply:
I'll pick this up in my next PR - it's a great suggestion. But now I have to make AMQP changes so I can merge AMQP into C++ :(.